### PR TITLE
LPS-147468 [CP 2.0] Fix downloaded key file name

### PR DESCRIPTION
**Pull request by joanagrego**

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/DXPActivationKeysTable/components/Header/index.js [hunk 1]
```diff
@@ -64,6 +65,22 @@ const DXPActivationKeysTableHeader = ({
 		return keyCanBeDownloaded;
 	}, [activationKeys, selectedKeys]);
 
+	const selectedKeysObjects = activationKeys.filter((item) => {
+		return selectedKeys.includes(item.id);
+	});
```
[the remark this answers is not in the document]
Comment (joanagrego) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/DXPActivationKeysTable/components/Header/index.js:70]: Nice! Done.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/DXPActivationKeysTable/components/Header/index.js [hunk 2]
```diff
@@ -64,6 +65,22 @@ const DXPActivationKeysTableHeader = ({
...
+
+	const selectedKeysProductNames = activationKeys
+		.filter((item) => {
+			return selectedKeys.includes(item.id);
+		})
```
[the remark this answers is not in the document]
Comment (joanagrego) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/DXPActivationKeysTable/components/Header/index.js:75]: Nice! Done.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/DXPActivationKeysTable/components/Header/index.js [hunk 3]
```diff
@@ -64,6 +65,22 @@ const DXPActivationKeysTableHeader = ({
...
+		.map((item) => item.productName);
+
+	const selectedKeysProductVersion = activationKeys
+		.filter((item) => {
+			return selectedKeys.includes(item.id);
+		})
```
[the remark this answers is not in the document]
Comment (joanagrego) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/DXPActivationKeysTable/components/Header/index.js:81]: Nice! Done.

**Comment by jairmedeiros** [comment]
ci:forward

**Review by jairmedeiros — APPROVED** [review]

**Pull request closed without merging**